src/project/view/cloudaudiofilesmodel.cpp

```
@@ -22,25 +23,25 @@ CloudAudioFilesModel::CloudAudioFilesModel(QObject* parent)
 
 void CloudAudioFilesModel::load()
 {
-    auto onUserAuthorizedChanged = [this](bool authorized) {
-        if (authorized) {
+    auto onUserAuthorizedChanged = [this](au::au3cloud::AuthState status) {
+        if (std::holds_alternative<au::au3cloud::Authorizing>(status)) {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)